package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "leader-board",
+  "version": "1.0.0",
+  "description": "A leader-board website where you can add new candidate, save their data to an API and also consume the data from the API",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
```

Comment:
- [ ] Please kindly add a script to run your project and ensure project is running before your resubmit for re-review. Something like so: 

`"start": "webpack --config webpack.config.js --watch"`

Reply:
DONE